### PR TITLE
Add UMA Token as a collateral asset

### PR DESCRIPTION
**Pull request by 0xtranspose**

### CODE REVIEW

File: markets/aave/commons.ts [hunk 1]
```diff
@@ -209,6 +210,7 @@ export const CommonsConfig: ICommonConfiguration = {
       SNX: '0xF9A76ae7a1075Fe7d646b06fF05Bd48b9FA5582e',
       SUSD: '0xb343e7a1aF578FA35632435243D814e7497622f7',
       TUSD: '0x7aeCF1c19661d12E962b69eBC8f6b2E63a55C660',
+      UMA: '0x489Bf230d4Ab5c2083556E394a28276C22c3B580',
```
Comment (miguelmtzinf) [inline markets/aave/commons.ts:213]: This should be the CL price feed (ETH denominated): 0xf817B69EA583CAFF291E287CaE00Ea329d22765C

File: markets/aave/commons.ts [hunk 2]
```diff
@@ -253,6 +255,7 @@ export const CommonsConfig: ICommonConfiguration = {
       SNX: '0x79291A9d692Df95334B1a0B3B4AE6bC606782f8c',
       SUSD: '0x8e0b7e6062272B5eF4524250bFFF8e5Bd3497757',
       TUSD: '0x3886BA987236181D98F2401c507Fb8BeA7871dF2',
+      UMA: '0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828',
```
Comment (miguelmtzinf) [inline markets/aave/commons.ts:258]: The address of the CL price feed (ETH denominated): 0xf817B69EA583CAFF291E287CaE00Ea329d22765C

File: markets/aave/commons.ts (hunk 1 above)
Comment (Tommy1231232) [reply markets/aave/commons.ts]: Deleted this line

File: markets/aave/commons.ts (hunk 2 above)
Comment (Tommy1231232) [reply markets/aave/commons.ts]: Updated to 0xf817B69EA583CAFF291E287CaE00Ea329d22765C

**Pull request closed without merging**